scripts/workflow-launcher.ps1

```
@@ -131,6 +131,44 @@ Write-Log "Log: $LOG_FILE"
+$QUOTA_FILE = Join-Path $WORKSPACE ".agent\quota_usage.json"
+$isCritical = $false
+if (Test-Path $QUOTA_FILE) {
+  try {
+    $quotaData = Get-Content $QUOTA_FILE -Raw | ConvertFrom-Json
+    $today = Get-Date -Format "yyyy-MM-dd"
+    if ($quotaData.date -eq $today -and $quotaData.count -gt 1350) {
+      # 90% of 1500
+      $isCritical = $true
+    }
+  }
+  catch {}
```

PSScriptAnalyzer (1.23.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)